contracts/nois-proxy/src/contract.rs

```diff
@@ -63,6 +70,8 @@ pub fn instantiate(
         nois_beacon_price_updated: Timestamp::from_seconds(0),
         mode,
         allowlist_enabled: Some(allowlist_enabled),
+        min_after: Some(env.block.time),
```

Comment:
I think the name here is not very descriptive especially that this config is going to be used by many people that wanna integrate the proxy. 
maybe 
- min_after -> manual_timestamp_mode_earliest_allowed_beacon_after
- max_after -> manual_timestamp_mode_latest_allowed_beacon_after
?

Reply:
What this checks in the current implementation is the `after` value of the request beacon IBC packet. So this applies for both ways to request the randomness. Just GetNextRandomness is unlikely to hit an error.

The values are set automatically on instantiation and usually never need to be changed. So the contact point is minimized. We still can try to expand the name but there are also docs on the field.